src/assets/GoogleIcon.tsx

```diff
@@ -1,10 +1,10 @@
 import React, { FunctionComponent } from 'react';
 import { SocialIconProps } from './types';
 
-export const GoogleIcon: FunctionComponent<SocialIconProps> = ({ width = 20, height = 20, viewBox = [0, 0, 48, 48] }) => <svg
-    width={`${width}`}
-    height={`${height}`}
-    viewBox={viewBox.join(' ')}
+export const GoogleIcon: FunctionComponent<Partial<SocialIconProps>> = ({ width = 20, height = 20 }) => <svg
```

Comment:
Jak mamy interface 

Button { 
a?: string;
b?: string;
c?: string;
d?: string;
}

to zamiast dawać wszędzie ? można usunąć je i otoczyć interface typem Partial<Button>. Jest to feature wzięty z typescriptu